ci/qemu/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Maybe `#!/usr/bin/env bash` in all files for consistency?

Reply:
done